README.md

```diff
@@ -11,16 +11,20 @@ To get a local copy up and running:
 
```

Comment:
- [ ] On your readme, please, use the [Microverse Readme Template](https://github.com/microverseinc/readme-template) to correct the formatting of the resulting text.   The headers and subheaders are not correctly formatted. 

Remember, all the sections are mandatory (except for "tests", in this case).

That way, your readme file will look professional 💪

Reply:
Bro I think I have followed that template. What needs to improve? I am not clear. Please I need help. I'll leave a message via slack.

Reply:
Hey @HtetNaing0814,

The formatting does not work:

![image](https://user-images.githubusercontent.com/57421823/172122896-5331e8f3-5210-455e-921f-d2a461a93dec.png)

Look at the formatting on the template:

![image](https://user-images.githubusercontent.com/57421823/172123008-dfd2aedb-c96c-40e0-9a7b-09747ee44623.png)

You can go [here](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md) and check how the markdown was used to properly format the readme and apply it to yours.

Don't worry, this is a one-time job because for the next projects you will be able to just copy-paste, and adapt the readme to the current project.

Reply:
Bro I just push it again. Is that ok now?